float64 x_pose
float64 y_pose
float64 w_pose

float64 x_trans
float64 y_trans
float64 z_trans

float64 x_rot
float64 y_rot
float64 z_rot
float64 w_rot
string state
